ctapipe/reco/ImPACT.py

```diff
@@ -93,20 +95,24 @@ class ImPACTReconstructor(Reconstructor):
     spe = 0.5  # Also hard code single p.e. distribution width
 
     def __init__(self, root_dir=".", minimiser="minuit", prior="",
-                 template_scale=1., xmax_offset=0, use_time_gradient=False):
+                 template_scale=1., xmax_offset=0, use_time_gradient=False,
+                 use_shower_variance=False):
 
         # First we create a dictionary of image template interpolators
         # for each telescope type
         self.root_dir = root_dir
         self.priors = prior
         self.minimiser_name = minimiser
 
-        self.file_names = {"CHEC": ["GCT_05deg_ada.template.gz",
-                                    "GCT_05deg_time.template.gz"],
+        self.file_names = {"CHEC": ["GCT_05deg_ada_ex.template.gz",
```

Comment:
it may be a good idea to make these Component configuration variables, or perhaps a configurable "pattern" like 

```py3
   template_pattern = "{cam_type}_05deg_{template_type}"
```
Where `template_type` is one of ["ada", "time", ...] 

Also what is the meaning of  "ada" and "ada_ex" ? could that be written more clearly in full words?

Reply:
Yes, I fully agree, this is a holdover from when the camera names kept changing. I will try to make this all a bit clearer.

Reply:
This could now be a TelescopeParameter